frontend/components/Contribute/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Heading } from '../shared/Heading';
+
+const WantToContribute = ({ data }) => {
+  const { description, img } = data;
+  return (
+    <div className="flex items-start justify-around">
+      <div>
+        <span className="block mt-16 py-5 uppercase">
+          <Heading type="cardSmall">Want To Contribute</Heading>
```

Comment:
`<Heading type="cardSmall">
            <GradientText gradientColor="blue" text="Want To Contribute" />
          </Heading>`
use the heading component this way to get the gradient effect